cli/pkg/local/app.go

```diff
@@ -186,6 +187,9 @@ func NewApp(ctx context.Context, opts *AppOptions) (*App, error) {
 	if opts.OlapDriver == DefaultOLAPDriver && olapDSN == DefaultOLAPDSN {
 		defaultOLAP = true
 		olapDSN = path.Join(dbDirPath, olapDSN)
+		// Set path to ensure that OLAP does not store data in runtime.Options.DataDir
```

Comment:
This is still the same as `DataDir`. Maybe note that otherwise it would be put in a nested directory and we need this for backwards compatibility.

Reply:
Sorry I didn't understand. `dbDirPath` is `<projectPath>/tmp`  and olapDSN is `<DefaultOLAPDSN>(main.db)` which is also the path to duckdb file currently.

Reply:
I just meant to clarify the comment. Right now it says "ensure that OLAP does not store data in runtime.Options.DataDir", but I think that is the opposite of what it does – because in the default case it is setting `DataDir == "./tmp"` and `olapDSN == "./tmp/main.db"`, right?